common/static/common/js/discussion/discussion_router.js

```diff
@@ -129,9 +128,7 @@
             DiscussionRouter.prototype.navigateToThread = function(thread_id) {
                 var thread;
                 thread = this.discussion.get(thread_id);
-                return this.navigate("" + (thread.get("commentable_id")) + "/threads/" + thread_id, {
-                    trigger: true
-                });
+                return this.showThread(thread);
```

Comment:
@itsjeyd Yeah, I'd rather understand the issue and do a proper fix in YONK-401 than use a workaround now and forget to fix it properly in the future. And we should be aiming to make this fork more similar to upstream, with a smaller diff.